src/_help/changes-management/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Changes management
+---
+
+- TOC
+{:toc}
+
+Tracking API changes is essential, both for users and developers.
+
+## What is a change
+
+Changes are differentiated into various categories:
```

Comment:
This doesn't explain what is a change. We should probably redefine the concept here.

Reply:
I'm not touching this part until it has been properly define on our end. My first draft was something around: "a change is a modification of your API definition with an impact on its behavior"

Reply:
Final version: A change is a modification of your API with an impact on its behavior

The change is done on the API (by changing its code). The definition just reflects this change, but when we talk about a change, it's an API one, not an API definition one, imo.